var/spack/repos/builtin/packages/palisade-development-git/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PalisadeDevelopmentGit(CMakePackage):
```

Comment:
Usually we don't append "git" to packages that can be downloaded from git. Sometimes packages have a mixture of archive versions and SCM versions, which can both be managed in the same package. Therefore I think this should be called `PalisadeDevelopment` (`Palisade` would also work - I don't think that's "taken").

Reply:
Done